source/native-plugins/midi-channel-ab.c

```diff
@@ -92,18 +91,18 @@ static const NativeParameter* midichanab_get_parameter_info(NativePluginHandle h
 
 static float midichanab_get_parameter_value(NativePluginHandle handle, uint32_t index)
 {
-    if (index > MAX_MIDI_CHANNELS)
-        return 0.0f;
+    if (index >= MAX_MIDI_CHANNELS)
+        return 0;
 
-    return handlePtr->channels[index] ? 1.0f : 0.0f;
+    return handlePtr->channels[index] ? 1 : 0;
 }
 
 static void midichanab_set_parameter_value(NativePluginHandle handle, uint32_t index, float value)
 {
-    if (index > MAX_MIDI_CHANNELS)
+    if (index >= MAX_MIDI_CHANNELS)
         return;
 
-    handlePtr->channels[index] = (value >= 0.5f);
+    handlePtr->channels[index] = (value >= 1);
```

Comment:
why did you change all floats to integers here?
the actual type is a float, so the *.0f makes sense, it is easy to understand when reading the code that the actual type is a float and not an integer.
The specific change that this comment replies to, please change it back to >= 0.5f.

Reply:
Ah, thank you. I'm not familiar with how to read C++.